packages/opencensus-core/src/tags/validation.ts

```diff
@@ -30,7 +30,7 @@ export function isValidTagKey(tagKey: TagKey): boolean {
 
 /** Determines whether the given String is a valid tag value. */
 export function isValidTagValue(tagValue: TagValue): boolean {
-  if (!tagValue) {
+  if (!tagValue || tagValue.value === null || tagValue.value === undefined) {
```

Comment:
Why add ` || tagValue.value === null || tagValue.value === undefined`? Because both `null` and `undefined` are falsey, wouldn't those be captured within the `!tagValue`?

Reply:
```!tagValue``` is not applicable to its property (``value```). Yes, both```null``` and ```undefined``` are falsey for value property.